spec/support/coverage_loader.rb

```diff
@@ -0,0 +1,26 @@
+MINIMUM_COVERAGE = 74
+
+unless ENV['COVERAGE'] == 'off'
+  require 'simplecov'
+  require 'simplecov-rcov'
+  require 'coveralls'
+  Coveralls.wear!
+
+  SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+    SimpleCov::Formatter::RcovFormatter,
+    Coveralls::SimpleCov::Formatter
+  ]
+  SimpleCov.start do
+    add_filter '/vendor/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.